File: programme/sum-of-numbers/sum-of-numbers.cs
// Write a programme to find sum of n numbers
using System;

namespace Codinasion.Programme
{
    public class SumOfNumbers
    {
        public static void Main()
        {
            Console.Write("Input  : ");
            String input = Console.ReadLine();
            string[] numbers = input.Split(" ");
            var sum = 0;
            foreach (var number in numbers)
            {
                int x = Int32.Parse(number);
                sum += x;
            }
            Console.WriteLine("Output : " + sum);
        }
    }
}

File: programme/sum-of-numbers/sum-of-numbers.js
// Write a programme to find the sum of n numbers

const input = "1 2 3 4";

function sumOfNumbers(numbers) {
  let sum = 0;
  for (let i of numbers.split(" ")) {
    sum += +i;
  }
  return sum;
}

console.log("Input  :", input);
console.log("Sum    :", sumOfNumbers(input));
